DESCRIPTION.md

```diff
@@ -9,6 +9,11 @@ Source code is also available at:
 
 # Release Notes
 
+- v1.4.2(Sep 19, 2022)
+
+  - Switch all source code to use f-strings and with statements
+  - SQLAlchemy jobs are not uploading test results to Tests table in snowhouse
```

Comment:
This is an internal change
```suggestion
```

Reply:
For 14, created a release note:

Improved reliability by always using context managers.